features/product/managing_products/sorting_products_within_a_taxon_by_position.feature

```diff
@@ -12,19 +12,137 @@ Feature: Sorting listed products from a taxon by position
         And this product is in "Soft Toys" taxon at 2nd position
         And the store has a product "Small pug"
         And this product is in "Soft Toys" taxon at 3rd position
+        And the store has a product "Tiny pug"
+        And this product is in "Soft Toys" taxon at 4th position
+        And the store has a product "Large pug"
+        And this product is in "Soft Toys" taxon at 5th position
+        And the store has a product "Medium pug"
+        And this product is in "Soft Toys" taxon at 6th position
+        And the store has a product "Giant pug"
+        And this product is in "Soft Toys" taxon at 7th position
+        And the store has a product "Mini pug"
+        And this product is in "Soft Toys" taxon at 8th position
+        And the store has a product "Nano pug"
+        And this product is in "Soft Toys" taxon at 9th position
+        And the store has a product "Micro pug"
+        And this product is in "Soft Toys" taxon at 10th position
+        And the store has a product "Puglet"
+        And this product is in "Soft Toys" taxon at 11th position
+        And the store has a product "Pug XL"
+        And this product is in "Soft Toys" taxon at 12th position
+        And the store has a product "Pug XS"
+        And this product is in "Soft Toys" taxon at 13th position
+        And the store has a product "Puggle"
+        And this product is in "Soft Toys" taxon at 14th position
+        And the store has a product "Pug Junior"
+        And this product is in "Soft Toys" taxon at 15th position
+        And the store has a product "Pug Senior"
+        And this product is in "Soft Toys" taxon at 16th position
+        And the store has a product "Pug Master"
+        And this product is in "Soft Toys" taxon at 17th position
+        And the store has a product "Pug Pro"
+        And this product is in "Soft Toys" taxon at 18th position
+        And the store has a product "Pug Expert"
+        And this product is in "Soft Toys" taxon at 19th position
+        And the store has a product "Ultimate Pug"
+        And this product is in "Soft Toys" taxon at 20th position
```

Comment:
I know it's a little bit too long, but I'm uncertain whether we should hide this, as it's pretty important.

Reply:
That's a lot of pugs

Reply:
![image](https://github.com/Sylius/Sylius/assets/80641364/08d3477e-6cab-4a11-8db1-4edb39d22b3d)

Reply:
I have mixed feelings about hiding this setup details. For me, it'll make reading harder (and figuring out why a given pug should be at a given position in the "then" steps).

In terms of "position - 1" I'll change it.

Reply:
Wdyt about this:
```
    Background:
        Given the store classifies its products as "Soft Toys"
        And the store has a product "Old pug" in the "Soft Toys" taxon at 0th position
        And the store has a product "Young pug" in the "Soft Toys" taxon at 1st position
        And the store has a product "Small pug" in the "Soft Toys" taxon at 2nd position
        And the store has a product "Tiny pug" in the "Soft Toys" taxon at 3rd position
        And the store has a product "Large pug" in the "Soft Toys" taxon at 4th position
        And the store has a product "Medium pug" in the "Soft Toys" taxon at 5th position
        And the store has a product "Giant pug" in the "Soft Toys" taxon at 6th position
        And the store has a product "Mini pug" in the "Soft Toys" taxon at 7th position
        And the store has a product "Nano pug" in the "Soft Toys" taxon at 8th position
        And the store has a product "Micro pug" in the "Soft Toys" taxon at 9th position
        And the store has a product "Puglet" in the "Soft Toys" taxon at 10th position
        And the store has a product "Pug XL" in the "Soft Toys" taxon at 11th position
        And the store has a product "Pug XS" in the "Soft Toys" taxon at 12th position
        And the store has a product "Puggle" in the "Soft Toys" taxon at 13th position
        And the store has a product "Pug Junior" in the "Soft Toys" taxon at 14th position
        And the store has a product "Pug Senior" in the "Soft Toys" taxon at 15th position
        And the store has a product "Pug Master" in the "Soft Toys" taxon at 16th position
        And the store has a product "Pug Pro" in the "Soft Toys" taxon at 17th position
        And the store has a product "Pug Expert" in the "Soft Toys" taxon at 18th position
        And the store has a product "Ultimate Pug" in the "Soft Toys" taxon at 19th position
        And I am logged in as an administrator
```

It's 2 times shorter and uses the "correct" position values.